src/main/resources/org/cbioportal/persistence/mybatis/GenePanelMapper.xml

```diff
@@ -121,6 +121,7 @@
         <include refid="selectGenePanelData"/>
         <include refid="fromGenePanelData"/>
         WHERE
+        SAMPLE_ID IS NOT NULL AND 
```

Comment:
@haynescd this is one we might want to undo because it changes profile counts.  this is what allows the system to recover from incomplete sample_profile table issue.